strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/storage/checksum/ChecksumCacheManager.java

```diff
@@ -27,200 +36,97 @@
  * remove the respective Checksum from the cachedChecksums).
  *
  * @author mtodorov
+ * @author Przemysław Fusik
  */
+@Component
 public class ChecksumCacheManager
+        implements DisposableBean
 {
 
-    private static Logger logger = LoggerFactory.getLogger(ChecksumCacheManager.class);
+    private static final String SHA1_KEY = "sha1";
```

Comment:
Hm... Could you perhaps use something like `EncryptionAlgorithmsEnum.SHA1.getAlgorithm()` here?

Reply:
done